source/usr/lib/mcvirt/iso.py

```diff
@@ -0,0 +1,158 @@
+# Copyright (c) 2015 - I.T. Dev Ltd
+#
+# This file is part of MCVirt.
+#
+# MCVirt is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# MCVirt is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with MCVirt.  If not, see <http://www.gnu.org/licenses/>
+
+import os
+import stat
+
+from mcvirt import MCVirtException
+from system import System
+
+class IsoManagement:
+    
+    @staticmethod
+    def checkName(name):
+        """Check that name ends in .iso"""
+        return name[-4:].lower() == '.iso'
+        
+    @staticmethod
+    def getFilename(path):
+        """Return filename part of path"""
+        return path.split('/')[-1]
+        
+    @staticmethod
+    def setIsoPermissions(path):

```

Comment:
Given that this function is within the 'Iso' class, I think naming this to simply 'setPermissions' would be good... which probably applies to getIsoList -> getList, addIso -> add, deleteIso -> delete etc :)